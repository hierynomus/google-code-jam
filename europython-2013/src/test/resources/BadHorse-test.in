4
1
Dead_Bowie Fake_Thomas_Jefferson
3
Dead_Bowie Fake_Thomas_Jefferson
Fake_Thomas_Jefferson Fury_Leika
Fury_Leika Dead_Bowie
4
John Dave
Bob Alice
Dave Mallory
Bob Mallory
10
Mephista New_Wave
Mephista Ursa
Zaladane Mai_Shen
Mephista Mai_Shen
White_Rabbit Hypnota
White_Rabbit New_Wave
Ursa Scandal
Zaladane New_Wave
Ursa Hypnota
Zaladane Scandal