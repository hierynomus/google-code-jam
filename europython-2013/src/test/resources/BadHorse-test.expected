Case #1: Yes
Case #2: No
Case #3: Yes
Case #4: No